apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/impl.ex

```diff
@@ -128,6 +128,11 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Impl do
 
     MessageTracker.ack_delivery(new_state.message_tracker, message_id)
     Logger.info("Device connected.", ip_address: ip_address_string, tag: "device_connected")
+
+    :telemetry.execute([:astarte, :data_updater_plant, :device_connection], %{}, %{
```

Comment:
I'd add a `:data_updater` level to all events:
`[:astarte, :data_updater_plant, :data_updater, :device_connection]`

This way we can differentiate them from other events we can add in other places